src/Base/BaseValidator.php

```
@@ -196,4 +196,34 @@ public static function positiveLike(
     }
+
+    /**
+     * Normalizes an allowed value to its exception message representation.
+     *
+     * @param mixed $value Allowed value.
+     *
+     * @return string Message value.
+     */
+    private static function normalizeAllowedValue(mixed $value): string
+    {
+        $normalized = Enum::normalizeValue($value);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Helper\Enum' in method 'normalizeAllowedValue'. (undefined)

(StaticAccess)